scripts/generate.sh

```diff
@@ -44,5 +44,5 @@ OUT_DIR=examples/java/src/main/java/com/twilio/rest
 generate twilio-java
 
 OUT_DIR=examples/node/lib/rest
-generate twilio-node
+generate twilio-node --global-property skipFormModel=false
```

Comment:
Isn't this already being set [here](https://github.com/twilio/twilio-oai-generator/pull/281/files#diff-53db1b6599656db8b4a898bdcc6632d6bbf39605ade0cb914f8f29c840927431R49)?

Reply:
This script is for building the examples. The other is for building the actual helpers.